run-tnf-container.sh

```diff
@@ -14,9 +14,10 @@
 #   KUBECONFIG=/usr/tnf/kubeconfig/config:/usr/tnf/kubeconfig/config.2
 
 export REQUIRED_NUM_OF_ARGS=5
-export REQUIRED_VARS=('LOCAL_KUBECONFIG' 'LOCAL_TNF_CONFIG' 'OUTPUT_LOC')
+export REQUIRED_VARS=('LOCAL_KUBECONFIG' 'LOCAL_TNF_CONFIG' 'OUTPUT_LOC' 'LOCAL_DOCKERCFG')
```

Comment:
Not sure whether I've understood this 100%, but, does this really need to be mandatory? If merged, that would force current partners using this tool to have a valid docker config file somewhere just to run the current test cases... can't we make it optional for now?

Reply:
I was thinking about this as well.  It's only required if you want to run the `preflight` suite of tests.  Should I make this completely optional and then just put a prompt in the logs that says if you are running `preflight` you must supply this variable?

Reply:
Yes, that makes more sense to me. I mean, right now the preflight PR is not merged yet, so adding now another mandatory flag for something that's not yet implemented seems weird to me.

Reply:
Sounds good I'll make this one optional.

Reply:
Addressed in https://github.com/test-network-function/cnf-certification-test/pull/804/commits/395a9d8c4f24b95972b461c5afedbba0887a767e